sccm/develop/reference/core/servers/configure/sms_advertisement-server-wmi-class.md

```diff
@@ -97,16 +97,20 @@ Class SMS_Advertisement : SMS_BaseClass
 |0x00000100 (8)|ONSYSTEMSTARTUP. Announce the advertisement to the user on system startup.|  
 |0x00000200 (9)|ONUSERLOGON. Announce the advertisement to the user on logon.|  
 |0x00000400 (10)|ONUSERLOGOFF. Announce the advertisement to the user on logoff.|  
+|0x00001000 (12)|OPTIONALPREDOWNLOAD. If the selected architecture and language matches that of the client, the package content will be downloaded in advance
 |0x00008000 (15)|WINDOWS_CE. The advertisement is for a device client.|  
-|0x00010000 (16)|ENABLE_PEER_CACHING <br /><br /> This information applies to System Center 2012 Configuration Manager SP1 or later, and System Center 2012 R2 Configuration Manager or later.|  
+|0x00010000 (16)|ENABLE_PEER_CACHING. This information applies to System Center 2012 Configuration Manager SP1 or later, and System Center 2012 R2 Configurcation Manager or later.|  
```

Comment:
oops, @ChrisKibble you have an extra "c" inserted here on Configuration

Reply:
Resolved in new commit.